lib/tasks/annual_fund_impact_metrics_activities.rake

```diff
@@ -7,36 +7,59 @@ namespace :activities do
 
     csv_path = args[:output_csv_path] || "tmp/annual_fund_impact_metrics.csv"
 
+    level_c_orgs = [
+      "Connected Places Catapult",
+      "Energy Systems Catapult",
+      "Offshore Renewable Energy Catapult",
+      "National Physics Laboratory",
+      "UK Atomic Energy Authority"
+    ]
+
     CSVSafe.open(csv_path, "wb") do |csv|
       activities = Activity
```

Comment:
Might be nice to have the `activities` var outside of the `CSVSafe` block to separate the query from the csv writing